modules/@apostrophecms/image-widget/views/widget.html

```diff
@@ -1,31 +1,39 @@
-{% if data.options.className %}
-  {% set className = data.options.className %}
-{% elif data.manager.options.className %}
-  {% set className = data.manager.options.className %}
-{% endif %}
+{% if apos.util.eqStrict(data.widget.aposPlaceholder, true) and data.manager.options.placeholderUrl %}
```

Comment:
This util.eqStrict check feels like overkill to me, especially in a template.

Reply:
I guess it's not wrong though so I won't flag it.

Reply:
Hmm, I see what you mean, I was wondering also..
But since I've used strict comparisons for `aposPlaceholder` [elsewhere](https://github.com/apostrophecms/apostrophe/pull/3882/files#diff-8afe4cc61ac0fbcc6476c7d670671b940a728f08619ef63f47ba0621323cc33aR78), I'd wanted to keep it consistent